src/components/TextEditor/TextEditor.jsx

```diff
@@ -0,0 +1,30 @@
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+import styled from 'styled-components';
+
+const modules = {
+  toolbar: {
+    container: [['bold', 'italic', 'underline'], [{ align: [] }], [{ list: 'bullet' }, { list: 'ordered' }]],
+  },
+};
```

Comment:
상수로 선언하면 대문자로 부탁드립니다. 추가로 해당 배열을 보기 좋게 줄내림을 하면 더 좋아요.
```suggestion
const MODULES = {
  toolbar: {
    container: [
      ['bold', 'italic', 'underline'], 
      [{ align: [] }], 
      [{ list: 'bullet' }, 
        { list: 'ordered' }
      ]
     ],
  },
};
```